### PR TITLE
Release 4.0.4

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.0.4 (2020-08-05)
+
+- Fixed the colorpicker component to not positions its overlay until all contents have fully rendered. [#62](https://github.com/blackbaud/skyux-colorpicker/pull/62)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md:3]: "to not position" the (s) is unnecessary.

Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: Does this work or change the meaning?
Fixed the colorpicker component to position its overlay after the contents fully render.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Updated to Denise's wording

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That's fine. Changed

**Pull request merged**